cli/src/cluster_query.rs

```diff
@@ -1786,6 +1794,26 @@ pub fn process_show_stakes(
             ]);
         }
     }
+
+    if let Some(withdraw_authority_pubkey) = withdraw_authority_pubkey {
+        // withdrawer filter
+        let withdrawer_filter = vec![rpc_filter::RpcFilterType::Memcmp(rpc_filter::Memcmp {
+            offset: 44,
+            bytes: rpc_filter::MemcmpEncodedBytes::Base58(withdraw_authority_pubkey.to_string()),
+            encoding: Some(rpc_filter::MemcmpEncoding::Binary),
+        })];
+
+        match program_accounts_config.filters {
+            Some(filters) => {
+                // filter by withdrawer
+                program_accounts_config.filters = Some([filters, withdrawer_filter].concat())
```

Comment:
This is a bit simpler:
```suggestion
            Some(ref mut filters) => {
                // add filter by withdrawer
                filters.push(withdrawer_filter);
```

Reply:
Ah, someone just merged this. Will follow-up myself.

Reply:
Ah crap, really sorry!

Reply:
Nbd! I actually realized we can get rid of the match altogether :) https://github.com/solana-labs/solana/pull/26038
Sadly, `get_or_insert_default` isn't stable yet